### PR TITLE
Update Simpleaf modules to 0.15.1

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
LGTM, provided the current tests work, if not, could you update them in nf-test tests instead of updating the pytest ones?

**Comment by apeltzer** [comment]
Yeah might need some TLC also as @rob-p pointed out there are new options available etc that we should probably expose to users. Can give it a try and then we can maybe work on adding the missing parameter(s) + description

File: modules/nf-core/simpleaf/index/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+                { assert snapshot(process.out.versions).match("versions") }
```
Comment (sateeshperi) [inline modules/nf-core/simpleaf/index/tests/main.nf.test:33]:
```suggestion
                { assert snapshot(process.out).match() }
```

Comment (sateeshperi) [reply modules/nf-core/simpleaf/index/tests/main.nf.test:33]: `process.out` will also capture the versions channel

File: modules/nf-core/simpleaf/quant/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    input[2] = Channel.of([ meta, transcriptome_fasta ])
+                    """
+                }
+            }
+        }
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
```
Comment (sateeshperi) [inline modules/nf-core/simpleaf/quant/tests/main.nf.test:35]:
```suggestion
```

Comment (sateeshperi) [reply modules/nf-core/simpleaf/quant/tests/main.nf.test:35]: don't need this param if not checking published files

**Pull request merged**